### PR TITLE
Javascript/javascript2/week3

### PR DESCRIPTION
**Pull request by Enqira**

### CODE REVIEW

**Comment by Enqira** [comment]
By mistake I did add 3 commit to js2 week3 that later I reverted

File: javascript/javascript2/week3/game/game.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+      label.innerHTML = "S is the WINNER!";
+    } else if (totalL > totalS) {
+      label.innerHTML = "L is the WINNER!";
+    } else {
+      label.innerHTML = "TIE!";
+    }
+    document.removeEventListener("keydown", pressedL);
+    document.removeEventListener("keydown", pressedS);
+  }, timeSet * 1000);
+}
```
Comment (rajanibusani) [inline javascript/javascript2/week3/game/game.js:58]: I tried to add a restart button to your game by using below code but...If You dont want just delete this comment enqira...
const btn2= document.getElementById("restart"); // added restart button in html
 /* here time,and label i made them global..;const time = document.getElementById("playTimeSet");
const label = document.getElementById("label");
and also i made a change  in this function..
function startGame() {
 const timeSet = time.value;
*/

function restartGame(){
  time.value = "";
label.textContent = "Play again";
totalS= totalL=0;
elS.textContent = "";
elL.textContent = "";
}
btn2.addEventListener("click", restartGame)*/

**Review by rajanibusani — COMMENTED** [review]
Hi Enqira...You did a very good job, and i tried my best to give to review ur code,But let me know if u think i made any mistakes...

File: javascript/javascript2/week3/game/game.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+  document.addEventListener("keydown", pressedL);
+}
+// counter for S
+
+let totalS = 0;
+const elS = document.querySelector("#countS");
+const pressedS = (e) => {
+  if (e.keyCode == 83) {
+    totalS++;
+    elS.innerHTML = totalS;
```
[the remark this answers is not in the document]
Comment (Enqira) [reply javascript/javascript2/week3/game/game.js:22]: Sure, thank you

**Pull request closed without merging**